redfish-core/lib/oem/ibm/pcie_topology_refresh.hpp

```diff
@@ -0,0 +1,154 @@
+#pragma once
+
+#include "async_resp.hpp"
+#include "dbus_utility.hpp"
+#include "redfish_util.hpp"
+
+#include <memory>
+#include <string>
+#include <variant>
+
+namespace redfish
+{
+// NOLINTBEGIN(cppcoreguidelines-avoid-non-const-global-variables)
+// Timer for PCIe Topology Refresh
+static std::unique_ptr<boost::asio::steady_timer> pcieTopologyRefreshTimer;
+static uint countPCIeTopologyRefresh = 0;
+// NOLINTEND(cppcoreguidelines-avoid-non-const-global-variables)
+
+/**
+ * @brief PCIe Topology Refresh monitor. which block incoming request
+ *
+ * @param[in] timer       pointer to steady timer which block call
+ * @param[in] asyncResp   Shared pointer for generating response message.
+ * @param[in] countPCIeTopologyRefresh   how many time this method get called.
```

Comment:
nit: These don't match the actual params.

Less of a nit, countPtr is also an out param.

Reply:
Changed